_CodingChallenges/142.1-rubiks-cube.md

```diff
@@ -27,6 +27,10 @@ contributions:
       name: "xxnavixx"
     url: "https://editor.p5js.org/xxnavixx/present/TyIwhGCVW"
     source: "https://github.com/xxnavixx/rubikcube.git"
+  - title: "Multi-sized cubes without internal colors"
+    author:
+      name: "Vladi Gotsin"
+    source: "https://github.com/CodingTrain/website/pull/1282"
```

Comment:
@vladig98 Thank you for this contribution. 
Unfortunately the other PR is not a valid code source. Please move your code to a dedicated repository of your choice.